Makefile

```diff
@@ -174,7 +174,7 @@ $(foreach DB,$(DATABASES),$(eval $(call database,$(DB))))
 
 ARTIX_PARTS=artix7_50t artix7_200t
 ZYNQ_PARTS=zynq7010
-KINTEX_PARTS=kintex70t
+KINTEX_PARTS=
```

Comment:
Can you explain why kintex70t was removed from here?

Reply:
This file does not exists in https://github.com/SymbiFlow/prjxray/tree/master/settings
After I added `db-extras-kintex7-parts: $(addprefix db-part-only-,$(KINTEX_PARTS))` in the Makefile, the CI loop fails.